core/templates/services/UpgradedServices.ts

```diff
@@ -686,6 +686,10 @@ interface UpgradedServicesDict {
 export class UpgradedServices {
   getUpgradedServices(): UpgradedServicesDict {
     var upgradedServices = {};
+    // We are using eslint disable here for multilines because we have not used
+    // dot notation at a lot of places so it is not possible to use
+    // eslint disable next line for each of them.
```

Comment:
Add the explanation that you have in the comment below (about what happens when you've changed the expression to dot notation). Don't just say "not possible" -- try not to use that phrase. Always explain why.